modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -44,3 +44,46 @@ where:
 --
 
 . Optional: If you need to encrypt traffic to external hosts, complete the "Configuring IPsec encryption for external traffic" procedure.
+.Verification
```

Comment:
I think you need another line before verification because on the preview this is coming up as a 3rd step.

Reply:
Does it make sense to have that heading like that in a procedure?

Reply:
hmmmm I am looking at guidelines because I have really only seen it as a heading, but I guess I don't see anything stating against it staying in the steps https://redhat-documentation.github.io/modular-docs/#verification

Reply:
I just added it as a heading.